python3/bin/perfmon

```diff
@@ -1472,35 +1476,28 @@ def main():
             for sr_mon in sr_mon_lookup.values():
                 sr_mon.process_rrd_updates(rrd_updates, session)
 
-        except socket.error as e:
-            if e.args[0] == 111:
-                # "Connection refused"
-                # this happens when we try to restart session and *that* fails
-                time.sleep(2)
-                pass
-
+        except ConnectionRefusedError as e:
+            # "Connection refused[111]"
+            # this happens when we try to restart session and *that* fails
+            time.sleep(2)
```

Comment:
The original logic would cover other `socket.error` also.
Only the "Connection refused[111]" requires `time.sleep(2)`.

Reply:
As I mentioned in the commit message, `socket.error` is now merged into `OSError`
The other exceptions of `socket.error` will be catched later in `OSError`.

Reply:
- In python3, socket.error, IOError are merged into OSError
- ConnectionRefusedError is a subclass of OSError:
  - ConnectionRefusedError -> ConnectionError -> OSError
- urllib.error.HTTPError is a subclass of OSError:
  - urllib.error.HTTPError <- urllib.error.URLError <- OSError
- HTTPError doesn't have content in `args`. So we can't use `e.args[0]`